### PR TITLE
ENDOC-796 version & minor edits

### PR DESCRIPTION
**Pull request by jyunmitch**

### CODE REVIEW

File: vuepress/docs/next/docs/reference/keycloak-cr.md [hunk 1]
```diff
@@ -17,11 +17,11 @@ spec:
   imageName: "entando/entando-keycloak"
   dbms: "postgresql"
   ingressHostName: "test-keycloak.192.168.1.1.nip.io"
-  entandoImageVersion: "7.0.0"
```
Comment (jyunmitch) [inline vuepress/docs/next/docs/reference/keycloak-cr.md:20]: Does this entandoImageVersion still belong?

**Review by antoalv19 — APPROVED** [review]

File: vuepress/docs/next/docs/reference/keycloak-cr.md (hunk 1 above)
Comment (firegloves) [reply vuepress/docs/next/docs/reference/keycloak-cr.md:20]: Looking in the current files of this type, it seems that it has been removed

File: vuepress/docs/next/docs/reference/keycloak-cr.md [hunk 2]
```diff
@@ -17,11 +17,11 @@ spec:
...
   tlsEnabled: false
   replicas: 1
-
  ```
+ <!-- does the entandoImageVersion belong? -->
```
Comment (firegloves) [inline vuepress/docs/next/docs/reference/keycloak-cr.md:23]: probably this comment is relative to the revies comment?

File: vuepress/docs/next/tutorials/devops/tls-with-cert-mgr.md [hunk 3]
```diff
@@ -16,9 +16,9 @@ kubectl create namespace cert-manager
 
 2. Follow the install instructions to create the `cert-manager` resources in this namespace. The install can be as simple as the following command if the default configuration is acceptable: 
 ``` bash
-kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.12.0/cert-manager.yaml -n cert-manager
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.14.0/cert-manager.yaml -n cert-manager
 ```
-
+<!-- Should this be changed to 1.14??? v1.12.0 requires app.kubernetes.io/version: "v1.12.0” while v1.14.0 requires v1.14.0 -->
```
Comment (firegloves) [inline vuepress/docs/next/tutorials/devops/tls-with-cert-mgr.md:21]: @pietrangelo can you please add something to this point?

File: vuepress/docs/v7.3/docs/reference/keycloak-cr.md [hunk 4]
```diff
@@ -17,11 +17,11 @@ spec:
   imageName: "entando/entando-keycloak"
   dbms: "postgresql"
   ingressHostName: "test-keycloak.192.168.1.1.nip.io"
-  entandoImageVersion: "7.0.0"
   tlsEnabled: false
   replicas: 1
-
  ```
+<!-- does the entandoImageVersion belong? -->
```
Comment (firegloves) [inline vuepress/docs/v7.3/docs/reference/keycloak-cr.md:23]: same as previous

File: vuepress/docs/v7.3/tutorials/create/pb/github-actions-workflow.md [hunk 5]
```diff
@@ -46,10 +46,10 @@ We'll now add a second job to the workflow definition, specifically for the micr
      runs-on: ubuntu-latest
      if: "!contains(github.event.head_commit.message, '[ci skip]') && !contains(github.event.head_commit.message, '[skip ci]') && !contains(github.event.pull_request.title, '[skip ci]') && !contains(github.event.pull_request.title, '[ci skip]')"
      steps:
-       - uses: actions/checkout@v2
-       - uses: actions/setup-node@v2.1.4
+       - uses: actions/checkout@v2 //does this need to be updated?
+       - uses: actions/setup-node@v2.1.4 //does this need to be updated?
```
Comment (firegloves) [inline vuepress/docs/v7.3/tutorials/create/pb/github-actions-workflow.md:50]: I think no, as previously specified

File: vuepress/docs/next/tutorials/devops/tls-with-cert-mgr.md (hunk 3 above)
Comment (jyunmitch) [reply vuepress/docs/next/tutorials/devops/tls-with-cert-mgr.md:21]: @firegloves just confirmed with Pietrangelo via Slack that this doesn't need to be updated

**Review by firegloves — APPROVED** [review]
LGTM

**Pull request merged**